blog/_posts/2022-08-08-managing-tunnels-with-argocd.md

```diff
@@ -0,0 +1,236 @@
+---
+layout: post
+title: Managing Inlets tunnels with Argo CD
+description: If you're managing multiple inlets tunnel servers, Argo CD could make your job simpler. We'll show you how to set up two inlets-pro TCP servers using the helm chart and SealedSecrets for the authentication tokens.
+author: Han Verstraete
+tags: management argocd kubernetes saas
+author_img: welteki
+image: /images/2022-managing-tunnels-with-argocd/background.png
+date: 2022-08-08
+---
+
+If you're managing multiple inlets tunnel servers, Argo CD could make your job simpler. We'll show you how to set up two inlets-pro TCP servers using the helm chart and SealedSecrets for the authentication tokens.
+
+## Introduction
+In the previous blog post, [How To Manage Customer Services From Within Your Product](https://inlets.dev/blog/2022/08/03/accessing-customer-services.html) we showed how to deploy multiple, isolated tunnels, one for each customer and service. Adding additional tunnels and managing them can become difficult at scale. Setting up a GitOps pipeline with [Argo CD](https://argo-cd.readthedocs.io/en/stable/) can help with automating these tasks. Argo CD lets you store the configuration for all your tunnels in a git repository. It synchronises the configuration to your cluster on a continual basis and makes sure all resources are up to date.
```

Comment:
We don't want to say that Inlets is difficult.

Invert the tone, saying that it can be made easier to manage at scale?